### DISCUSSION TITLE
Remove broken rooms in multi

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#22098] [Q&A] [answered]
This rooms stay open and active for more than a month. They're locked and unaccessable (the first one has "123" password but gives an error when trying to join)
![image](https://user-images.githubusercontent.com/86934170/211356602-10d37874-4e33-437e-8d38-0ef1ee2e57ab.png)
![image](https://user-images.githubusercontent.com/86934170/211357009-0916b0ec-b7fb-4fe9-9bf7-bc5e7b1cbc65.png)

**Comment by peppy** [comment, marked as answer]
Thanks, these have been removed.
